src/compiler/toitdoc_parser.cc

```diff
@@ -873,6 +873,11 @@ std::string ToitdocParser::make_string(int from, int to) {
       if (i >= to) break;
     }
     int c = text[i];
+    if (c == '\r' && i < to - 1 && text[i + 1] == '\n') {
```

Comment:
Back in the day, you would also need to collapse \n\r to \n.

Reply:
Fortunately these machines have died out...